src/components/createNewCqlLibrary/CreateNewCqlLibrary.test.tsx

```diff
@@ -187,6 +278,15 @@ describe("Create New Cql Library Component", () => {
     ) as HTMLInputElement;
     userEvent.type(input, "TestingLibraryName12");
     expect(input.value).toBe("TestingLibraryName12");
+    const modelDropdown = screen.getByRole("button", {
+      name: /select a model/i,
+    });
+    userEvent.click(modelDropdown);
+    const qiCoreOption = screen.getByText("QI-Core");
+    expect(qiCoreOption).toBeInTheDocument();
+    userEvent.click(qiCoreOption);
+    const qiCore = await screen.findByText("QI-Core");
+    expect(qiCore).toBeInTheDocument();
```

Comment:
May be we can reduce constants by adding await inside the expect itself.

Reply:
we could, I think I was using more verbose language to try to improve readability. I was also trying to follow guides recommended by [Kent C Dodds here](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library) and at first he starts out with them separate but then switches to a mix of defining the constants on one line and performing assertions on the next line, and doing it all on one line.
So, I don't know really haha

Reply:
I renamed this variable to provide clarity over what makes it different from the previous search and click

Reply:
make sense, Good to know. very Interesting things in that article. I believe I did used some of the items he mentioned not to use. haha !!